modules/users/server/controllers/users/users.authentication.server.controller.js

```diff
@@ -83,18 +83,17 @@ exports.signout = function (req, res) {
 /**
  * OAuth provider call
  */
-exports.oauthCall = function (strategy, scope) {
-  return function (req, res, next) {
+exports.oauthCall = function (req, res, next) {
+    var strategy = req.params.strategy;
     // Authenticate
-    passport.authenticate(strategy, scope)(req, res, next);
+    passport.authenticate(strategy)(req, res, next);
```

Comment:
Integrating passport-azure-ad is a good idea. I'll be looking forward to seeing that come in.

We're just about ready to release 0.5.0. Which should be in the coming days.

We'll get this merged into *master* shortly after our release.